scripts/restore-sanitised-database.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+usage() {
+    echo "Usage: $0 S3 BUCKET NAME "
+    echo
+    echo "Arguments:"
+    echo " S3 BUCKET NAME"
+    echo
+    echo "Example:"
+    echo "  $0 aws-s3-bucket-sanitise-database"
+    echo "  This sets custom bucket name"
+    echo "  $0 "
+    echo "  This uses default bucket name"
+    exit 1
+}
+
+## Check if help was requested
+if [[ $1 == "--help" || $1 == "-h" ]]; then
+    usage
+fi
+
+# Default values for variables
+DEFAULT_S3_BUCKET=""
```

Comment:
Can we remove this default for now? As we don't have one and if people try to use this it'd break currently?
So make it mandatory but keep the help text?

Reply:
Done!